### ISSUE TITLE
请教一下训练时候为什么不适用softmax

### ISSUE THREAD

**Issue by DrewdropLife** [PaddlePaddle/PaddleOCR#5159]
我看识别模型的训练代码中，
        if not self.training:
            predicts = F.softmax(predicts, axis=2)
            result = predicts
训练阶段为什么不需要使用softmax呢？这样损失不是很容易爆炸吗，请教一下各位大佬。

**Comment by ziyaxuanyi** [comment]
训练使用的ctc-loss调用了baidu-research::warp-ctc的实现，该函数里面已经包含了softmax操作。
